src/controllers/users/createCart.ts

```diff
@@ -180,24 +180,23 @@ export default [
         return badRequest(response, ResponseError.EmptyBasket);
       }
 
+      if (
+        cartItems.reduce(
+          (previous, current) => previous + (current.reducedPrice ?? current.price) * current.quantity,
+          0,
+        ) < 0
+      ) {
+        return forbidden(response, ResponseError.BasketCannotBeNegative);
+      }
```

Comment:
Can you add explanation and why is there a "," at line 186 ?

Reply:
eslint ^^